### PR TITLE
feat(icons): add logout icon

### LINKED ISSUE #505: Add Logout icon
## 🚀 Feature Proposal
Add an icon called `logout` to the Icon set. 

It should use the font awesome sign out icon: https://fontawesome.com/icons/sign-out-alt?style=solid

### PR DESCRIPTION
**Pull request by morrme**
Fixes #505 

**Changes proposed in this pull request:**
added a `logout` icon

**Newly added dependencies with [Bundlephobia](https://bundlephobia.com/) links:**
- none

### CODE REVIEW

File: src/components/Icon/Icon.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ const iconMap = {
   incident: 'file-alt',
   lab: 'microscope',
   'left-arrow': 'chevron-left',
+  logout: 'sign-out-alt',
```
Comment (jackcmeyer) [inline src/components/Icon/Icon.tsx:22]: we should also include a test and a storybook example

**Comment by morrme** [comment]
@jackcmeyer Is there something I need to fix related to this change?

**Comment by jackcmeyer** [comment]
> @jackcmeyer Is there something I need to fix related to this change?

yes. you should be able to run `npm run lint` to identify what the error is. And then, follow the error message to resolve it.

**Comment by morrme** [comment]
@jackcmeyer  Now there is a story issue...any ideas?

**Comment by morrme** [comment]
@tehKapa Are there additional changes that need to be made?

**Comment by tehKapa** [comment]
> @tehKapa Are there additional changes that need to be made?

No, let's wait for all builds are done

**Comment by morrme** [comment]
graz x1000 @tehKapa

**Pull request merged**